### PR TITLE
Unit Notes Added

### PR DESCRIPTION
**Pull request by Natasha-H-S**
I have added a unite tile to the homepage. This Unit page, includes google drive account. This allows for apprentices to upload their work,and view the notes on different units. 

This has tests written and all current tests now pass.

### CODE REVIEW

File: test/components/Global/NavBar.test.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ describe('<NavBar />', () => {
   });
 
   it('shows 5 nav items', () => {
```
Comment (Showard180) [inline test/components/Global/NavBar.test.js]: can you change text to 6

File: test/components/Home/Home.test.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ describe('<Home />', () => {
   });
 
   it('renders 4 tiles', () => {
-    expect(component.find('Tile').length).toEqual(4);
+    expect(component.find('Tile').length).toEqual(5);
```
Comment (Showard180) [inline test/components/Home/Home.test.js]: change text to 5

File: test/components/Unit/unit.test.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```
Comment (Showard180) [inline test/components/Unit/unit.test.js]: Could file name be the same as component file name `Units/Units.test.js`

File: build/index.html [hunk 4]
```diff
@@ -1 +1 @@
-<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Apprentice Homepage</title><link href="/static/css/main.e91fd36f.css" rel="stylesheet"></head><body><div id="root"></div><script type="text/javascript" src="/static/js/main.30422a99.js"></script></body></html>
\ No newline at end of file
+<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Apprentice Homepage</title><link href="/static/css/main.e91fd36f.css" rel="stylesheet"></head><body><div id="root"></div><script type="text/javascript" src="/static/js/main.659a0bd9.js"></script></body></html>
```
Comment (Showard180) [inline build/index.html]: Could you add `build/` to `.gitignore` please, don't want to be commiting the built version

File: package.json [hunk 5]
```diff
@@ -24,6 +24,7 @@
     "body-parser": "^1.17.1",
     "express": "^4.15.2",
     "express-session": "^1.15.2",
+    "googleapis": "^18.0.0",
```
Comment (Showard180) [inline package.json]: Also this doesn't seem to be used anywhere, you possibly used it for the Api stuff and forgot to remove?

Comment (Natasha-H-S) [inline package.json]: Yh, I left in, in case you wanted to pick up. Removed now. :)

File: build/index.html (hunk 4 above)
Comment (Natasha-H-S) [inline build/index.html]: This is already in git ignore. -> Confused, due to me making no changes and this not showing when committing.

File: src/components/Units/Units.jsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+      <div className="o-container">
+        <div className="o-layout">
+          <div className="o-layout__item">
+            <article className="c-tile">
+              <div className="c-tile__content">
+                <div className="c-tile__body c-tile__white">
+                  <p className="c-heading-alpha u-margin-bottom">Unit Notes</p>
+                  <hr className="c-divider" />
+                  <div className="c-text-body">
+                  <p><a className="t-link-drive" href="https://drive.google.com/drive/folders/0B-0Qr07YB93sUlBzcXpkdFRBSEk?usp=sharing" className="c-link-external">Link To The Drive Here</a></p>
```
Comment (Showard180) [inline src/components/Units/Units.jsx]: Does `t-link-drive` class do anything

And can we get rid of the whitespace between `render` and `const src`

Also can we bring the href for the `a` tag into a variable please

File: build/index.html (hunk 4 above)
Comment (Showard180) [inline build/index.html]: Yeah i don't know why it is there, i don't think your's needs to be in the gitignore, maybe it was put there when you did an update from develop or something

**Pull request merged**